File: online-judge-solutions/Codeforces/1220A.cpp
//https://codeforces.com/problemset/problem/1220/A
//Lang		:	GNU C++14
//Time		:	30 ms
//Memory	:	300 KB
#include<iostream>
#include<algorithm>
using namespace std;
int main(){
	int n;
	string s;
	while(cin>>n>>s){
		int nzero=count(s.begin(),s.end(),'z');
		int nuno=count(s.begin(),s.end(),'n');
		for(int i=0;i<nuno;i++){cout<<1<<" ";}
		for(int i=0;i<nzero;i++){cout<<0<<" ";}
		cout<<endl;
	}
	return 0;
}

File: online-judge-solutions/Codeforces/1220A.py
#https://codeforces.com/problemset/problem/1220/A
#Lang		:	Python 3.8
#Time		:	62 ms
#Memory     :	200 KB
n,s=int(input()),input()
nzero=s.count('z')
nuno=s.count('n')
for i in range(nuno+nzero):
    print(1,end=" ") if i<nuno else  print(0,end=" ")
